guides/v2.3/javascript-dev-guide/widgets/widget_password_strength_indicator.md

```diff
@@ -0,0 +1 @@
+guides/v2.2/javascript-dev-guide/widgets/widget_password_strength_indicator.md
```

Comment:
The syntax for a symbolic link needs to be:

../../../../guides/v2.2/javascript-dev-guide/widgets/widget_password_strength_indicator.md